indico/htdocs/js/indico/modules/timetable/timetable/Base.js

```diff
@@ -1286,21 +1286,19 @@ type("TopLevelManagementTimeTable", ["ManagementTimeTable", "TopLevelTimeTableMi
         if (updateCycle) {
             updateCycle(data);
         } else {
-
             // If none is defined in the function args,
             // execute the default action
-            if(exists(result.slotEntry)){
+            if (exists(result.slotEntry)){
                 data[result.day][result.slotEntry.id].entries[result.entry.id] = result.entry;
-            }
-            else {
-                data[result.day][result.id] = result.entry;
+            } else {
+                data[result.day][result.entry.id] = result.entry;
```

Comment:
You **probably** won't need to modify this if you use `serialize_entry_update` to wrap the response.